src/hooks/useSections.ts

```diff
@@ -56,6 +56,14 @@ export default function useSections(
     recommendationsSections
   );
 
+  // Delete active sections if feature toggle, zeroStateActiveSessions is present
+  if (zeroStateActiveSections && podsData && Object.keys(podsData).length) {
+    const features = getSearchSuggestionFeatures(Object.values(podsData)[0]);
+    if (features.featureDisplayZeroStateRecommendations) {
+      activeSections.length = 0;
```

Comment:
Not too sure I like what I'm doing here.

Reply:
there could be side effects when setting `activeSections.length = 0` since it modifies the original array. you might want to have immutability for cleaner and predictable code. 

consider this scenario
const sections = [a,b,c];
console.log(sections) // prints [a,b,c] 
const {...} = useSections(query, cioClient, sections);
// it runs `activeSections.length = 0`
console.log(sections) /// prints [] 


off the top of my head, you could instead do 
const [activeSections, setActiveSections] = useState(zeroStateActiveSections ? zeroStateSections : sections);
useEffect(() => {
  // Delete active sections if feature toggle, zeroStateActiveSessions is present
  if (...) {
    setActiveSections([]);
  }
}, [zeroStateActiveSections, podsData, getSearchSuggestionFeatures])

Reply:
Thanks, that is definitely conforms to the react pattern better.